### DISCUSSION TITLE
Upgrade error/mistake

### DISCUSSION THREAD

**Discussion by HakanKoseoglu** [magicbug/Cloudlog#1675] [General]
Post-upgrade, it says there are no logbooks configured.

![image](https://user-images.githubusercontent.com/10955987/195910443-638d6ad4-9af0-420b-ab2c-c4aa28356fb3.png)

On the other hand... 

![image](https://user-images.githubusercontent.com/10955987/195910627-e2741034-2132-47a1-9df4-7916381bb0ee.png)

and

![image](https://user-images.githubusercontent.com/10955987/195910717-dc42035b-a32e-4142-92ba-b881d9d7684c.png)

Pretty sure my old location was M0KHR, because all of the exported ADIF logs are as such, not a big loss, I guess I can import it all but I must have taken a wrong turn at some point.

Running on my Docker container, freshly baked, against my old, upgraded config files.

**Comment by HakanKoseoglu** [comment]
Oh dear, replicated the same problem with my wife's logbook install. All stations claimed, clicked on the button to verify, logged in and the same problem.

**Reply by HakanKoseoglu** [reply]
Slowly figuring out what went wrong. The first user wasn't my user. I had deleted those, and the first user was a spare user on the system. So I need to figure out a way of associating the old logbooks back with the user I want.

![image](https://user-images.githubusercontent.com/10955987/195919492-b6ed9009-db82-44be-99c5-8025d380d027.png)

**Reply by HakanKoseoglu** [reply]
Resetting the station_profile as below fixed most of the issues:

update station_profile set user_id=6 where user_id=7;

Now off to fix my wife's logbook. :)

**Reply by HakanKoseoglu** [reply]
Done! That was easier (and the same problem).

**Comment by magicbug** [comment]
Yes sadly it was the only safe method was to take first user, I knew it would affect a few but crossed fingers.

**Reply by AndreasK79** [reply]
Why not use id from the logged in user?

**Reply by magicbug** [reply]
Because user may not have been logged in, I mean could hide it after the sign in.

**Reply by AndreasK79** [reply]
Would think that would be a safer option. Since I didn't study the code, I am just thinking out loud.

**Reply by HakanKoseoglu** [reply]
I think an order by would be useful when fetching the users, my user id was the lowest available and was 6, but MySQL returns my user_id of 7 first.

**Comment by rayslinky** [comment]
The gui shows my station locations, but no logbooks. If i look in the db, all my data is still in TABLE_HRD_CONTACTS_V01, however I don't have anything in station_logbooks or station_logbooks_relationship. Is there a way to force rerun the migration?

The user_id in my station_profile is correct.

**Comment by TheMcBain92** [comment]
Any Advice resolving this issue. Read though the current replies and not fully understanding what the cause is. From memory I've only ever had one user on my log and one location Just updated to V2 clicking the claim buttons etc.. and getting the same "No Logbooks were found error"

I can view my station location and it says there is QSO's there...

![2022-10-15 19_32_26-Station Location - Cloudlog](https://user-images.githubusercontent.com/27852607/196002629-1afa418f-4df1-4715-9528-254dc670db1e.png)

**Reply by TheMcBain92** [reply]
Update.

Manually created an Station Logbook and Linked it to the location above and all QSO's are now listed.

**Reply by magicbug** [reply]
Go to Station Logbooks in the drop-down create one and assign locations to it

**Reply by TheMcBain92** [reply]
Yes. Just found this. Did I manage to read past this on the V2 upgrade instructions? anyway... all working now Thanks
